res/controllers/common-controller-scripts.js

```diff
@@ -315,6 +315,10 @@ script.spinback = function(channel, control, value, status, group) {
    Output:  none
    -------- ------------------------------------------------------ */
 script.brake = function(channel, control, value, status, group, factor) {
+    // set default factor
+    if (!factor) {
```

Comment:
This will not work as you expect because of JavaScript's type coercion. Use `factor === undefined`